app/components/task_list/task_header.rb

```diff
@@ -0,0 +1,34 @@
+class TaskList::TaskHeader < ViewComponent::Base
+  def initialize(project:, task:)
+    @locales_path = task.locales_path
+    @title = title
+    @establishment_name = project.establishment.name
+    @hint = hint
+    @guidance_link = guidance_link
+    @guidance = guidance
+  end
+
+  private def title
```

Comment:
Can these just be public methods rather than assigning their values to instance variables?

Reply:
I don't think so? View components work like views, they need to be instance variables or helper methods?

Reply:
I'll pull this down and check, I'm half sure they can

Reply:
The section one looks like it can.

This helper `task_status_tag(task)` looks like it is used for both new and old as well so needs work.

Reply:
SImplest thing to do right now is to keep `title` on `TaskList::Task` and remove it later?

Reply:
I think we should probably rework the `task_status_tag(task)` method. The new implementation is relying on the old one being there.

`task_status_tag` calls `task_status_id` which is the one that requires a name